src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,213 @@
+package duke.ui;
+
+import duke.task.Task;
+import duke.task.TaskList;
+
+
+/**
+ * Class for handling UI (user facing interface)
+ */
+public class Ui {
+    private final String UI_LOGO = "             _____             _____   _    _              _   _ \n"
+            + "     /\\     |_   _|           / ____| | |  | |     /\\     | \\ | |\n"
+            + "    /  \\      | |    ______  | |      | |__| |    /  \\    |  \\| |\n"
+            + "   / /\\ \\     | |   |______| | |      |  __  |   / /\\ \\   | . ` |\n"
+            + "  / ____ \\   _| |_           | |____  | |  | |  / ____ \\  | |\\  |\n"
+            + " /_/    \\_\\ |_____|           \\_____| |_|  |_| /_/    \\_\\ |_| \\_|\n"
+            + "                                                                 \n"
+            + "                                                                 ";
+
+    private final String GUI_LOGO =
+                      "              _____             _____   _    _              _   _       \n"
+                    + "     /\\       |_   _|           / ____| | |  | |     /\\      | \\ | |        \n"
+                    + "    /  \\        | |    ______  | |      | |__| |    /  \\     |  \\| |        \n"
+                    + "   / /\\ \\      | |   |______| | |       |  __  |   / /\\ \\    | . ` |       \n"
+                    + "  / ____ \\   _| |_           | |____  | |  | |  / ____ \\   | |\\  |          \n"
+                    + " /_/    \\_\\ |_____|         \\_____| |_|  |_| /_/    \\_\\ |_| \\_|          \n"
+                    + "                                                                               \n"
+                    + "                                                                                 ";
+    private final String LINE_BREAK = "\n\t ^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^\n";
+    /**
+     * Default constructor.
+     */
+    public Ui() {
+    }
+
+    /**
+     * Method for first greet message.
+     */
+    public void greet() {
```

Comment:
Since you have two sets function for gui and non-gui, would it be clearer if you specificed the context of the non-gui functions in their names? (ie. defaultGreet, defaultDeleteMessage, etc)